templates/Engine method templates/BHoMObjectTemplate/MyTemplate.vstemplate

```diff
@@ -0,0 +1,21 @@
+<VSTemplate Version="3.0.0" xmlns="http://schemas.microsoft.com/developer/vstemplate/2005" Type="Item">
+  <TemplateData>
+    <DefaultName>BHoMObjectTemplate.cs</DefaultName>
+    <Name>BHoMObject - Add a new class</Name>
+    <Description>Adds a BHoMObject to your oM project.</Description>
+    <ProjectType>CSharp</ProjectType>
+    <SortOrder>10</SortOrder>
+    <Icon>CreateBHoM.ico</Icon>
+  </TemplateData>
+  <TemplateContent>
+    <References>
+      <Reference>
+        <Assembly>BHoM</Assembly>
```

Comment:
Can we not hardcode the hint path to the `C:/ProgramData/BHoM/Assemblies` folder or is that not possible?

Reply:
This is for an item, not a project. From memory, this is just stating "This item template requires BHoM to be used by this project"

Reply:
Ok, am happy